src/libraries/System.Linq/tests/default.rd.xml

```diff
@@ -0,0 +1,303 @@
+<Directives>
+  <Application>
+    <Assembly Name="System.Linq.Tests">
+      <Type Name="System.Linq.Tests.MinTests">
+        <Method Name="MinBy_Generic_RunOnce_HasExpectedOutput" Dynamic="Required All">
```

Comment:
Having to duplicate all of the test names into this default.rd.xml file seems very fragile.  Anyone refactoring / renaming / etc. these tests is very likely to miss the fact that this file will also need to be updated.  There's no better way to achieve the same goal?

Reply:
Maybe we can do a have dummy dynamically unreachable call in `WrapArgs` to ensure sure that the right method instantiations are available at runtime?

Reply:
The ultimate fix is to stop using generic methods as `Fact`/`Theory` since it's unsupportable with source generated runners where we would eventually like to go to (except maybe some subset that can be statically analyzed - xunit does a lot of runtime conversions for these).

If this is blocking, I'm honestly just going to create an issue to bring up System.Linq and keep the test disabled. This file took forever to make and I just want it to be over for my own sanity. I still have 80 test assemblies to go over for .NET 7 and this test is low value - the only value it has is extra random code to compile and run, but there are no NativeAOT-specific code paths